ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -137,18 +137,18 @@ class MySqlNamedBlobDb implements NamedBlobDb {
    */
   private static final String INSERT_QUERY =
```

Comment:
For the old blob, I think you probably can do the original way, and we only take care of get/list request.

Reply:
I feel for existing ones we handle in get/list is good. For new data insert, let's try it out in the new style (deleted_ts only).

Reply:
yeah, the reason I recommend to not change put for old named blob DB is if something wrong, for the old named blob, we still have the delete_ts and expire_ts data that we can refer. Since we don't need to change any schema in the old DB.

Reply:
Sounds good, just updated. So the old table's get/list request has change, the other requests stay same.